lib/configuration/configuration.js

```diff
@@ -86,10 +88,30 @@ class Configuration {
     }
   }
 
+  static async updateCache (context, cache) {
```

Comment:
I find the name of this function to be confusing. It is reseting/clearing the cache here and not necessarily updating the cache. The set occurs in `fetchConfigFile`. I believe it will be clearer to name this function something like `resetCache` 

Thoughts?

Reply:
Done